library/logrotate.py

```diff
@@ -199,6 +214,12 @@ def generate_config(module):
     options = []
     if module.params.get('compress'):
```

Comment:
iterate over options_with_argument and options_with_noargument instead of repeated 'if' does not work?